Instagram/View Controllers/PostDetailsViewController.m

```diff
@@ -0,0 +1,55 @@
+//
+//  PostDetailsViewController.m
+//  Instagram
+//
+//  Created by Sarah Wang on 7/7/21.
+//
+
+#import "PostDetailsViewController.h"
+#import "Post.h"
+#import "DateTools.h"
+
+@interface PostDetailsViewController ()
+
+@property (weak, nonatomic) IBOutlet UIImageView *photoImageView;
+@property (weak, nonatomic) IBOutlet UILabel *usernameLabel;
+@property (weak, nonatomic) IBOutlet UILabel *createdAtLabel;
+@property (weak, nonatomic) IBOutlet UILabel *postCaptionLabel;
+
+@end
+
+@implementation PostDetailsViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    [self refreshData];
+}
+
+- (void)refreshData {
```

Comment:
nit: Might call this `configureViews` or something. The fetching the post makes it a little more reasonable to call it a refresh but it's a bit misleading imo.